2023/Nwajei Edgar/README.md

```diff
@@ -1,19 +1,92 @@
+
+- [x] Write a short blog post which demonstrates your understanding of **accessibility**. In English, with at least one image, less than 500 words. Demonstrate your understanding _based on past experiences_. For example, any specific accessibility improvements you’ve done in the past, or specific issues you identified. The more specific the better. The post has to be posted on a publicly-available platform (Dev.to, Hashnode, Medium, your own website), and you must also add it as a new section in your personal file in this repository (so we can provide feedback on the contents).
+- [x] Write a short blog post which demonstrates your expertise in **technical writing**. In English, with at least one image, less than 500 words. Demonstrate your expertise _based on past experiences_. For example, any specific documentation writing you’ve done in the past, or other technical writing you’ve made. The more specific the better. The post has to be posted on a publicly-available platform (Dev.to, Hashnode, Medium, your own website), and you must also add it as a new section in your personal file in this repository (so we can provide feedback on the contents).
+- [] Ask [@thibaudcolas](https://github.com/thibaudcolas) via email or on Slack to give you editor access to a Wagtail website. Once you’ve received access, we ask you to create a new page demonstrating some of Wagtail’s accessibility features.
+- [ ] With your access to the demo site and by reviewing our [user guide](https://guide.wagtail.org/), identify a feature of Wagtail which isn’t documented, and report it back on the [guide project issue tracker](https://github.com/wagtail/guide/issues). Make sure to explain what the feature is, why and how you think it should be documented.
+- [ ] Draft the new content to document the feature of Wagtail which you’ve reported isn’t documented. Add it to the issue you created as a new comment, so we can provide feedback.
+- [ ] Ask [@thibaudcolas](https://github.com/thibaudcolas) via email or on Slack to give you editor access to [guide.wagtail.org](https://guide.wagtail.org/). Once you’ve received access, we ask you to make the necessary changes in the CMS to add the feature documentation.
+- [ ] TBC – Additional tasks specific to the Outreachy application process.
+
+
+
+
+
+
+
 ## Research
 - [Wagtail user guide](https://guide.wagtail.org/en-latest/)
 - [Official Wagtail website](https://wagtail.org/)
+
+## Recent learning.
+
+Short blog post on what i learnt recently https://0xwind.hashnode.dev/my-tech-trinity
+
+### Accessibility features documentation
+Short blog post on accessibility: https://0xwind.hashnode.dev/enhancing-user-experience
+
+
+## technical writing.
+Short blog post on technical writing : https://0xwind.hashnode.dev/my-javascript-journey
```

Comment:
Again a very generic article. I want to read things that demonstrate your understanding based on your experience, not easily-googleable information.